src/core/Expander/ExpanderGroup/ExpanderGroup.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv, HtmlButton, HtmlButtonProps, HtmlSpan } from '../../../reset';
+import { baseStyles } from './ExpanderGroup.baseStyles';
+import { VisuallyHidden } from '../../../components';
+
+const baseClassName = 'fi-expander-group';
+const openClassName = `${baseClassName}--open`;
+const expandersContainerClassName = `${baseClassName}_expanders`;
+const openAllButtonClassName = `${baseClassName}_all-button`;
+interface InternalExpanderGroupProps {
+  /** Expanders and option other components */
+  children: ReactNode;
+  /** 'Open all' button text */
+  OpenAllText: string;
+  /** 'Close all'-component (Button) */
+  CloseAllText: string;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** 'Open all' button text for screen readers, hides OpenAllText for screen readers if provided */
+  AriaOpenAllText?: string;
+  /** 'Close all' button text for screen readers, hides CloseAllText for screen readers if provided */
+  AriaCloseAllText?: string;
+  /** Open/Close all button props */
+  toggleAllButtonProps?: Omit<
+    HtmlButtonProps,
+    | 'onClick'
+    | 'onMouseDown'
+    | 'onMouseUp'
+    | 'onKeyPress'
+    | 'onKeyUp'
+    | 'onKeyDown'
+  >;
```

Comment:
Would it be more clear if this is extracted outside of the interface?

Reply:
It does not matter documentation wise (tried it out) and this type is not used elsewhere. Moving it out from the interface increases the amount of code by a little and doesn't provide any additional value in this case, at least in my opinion.

Reply:
Then let's leave it as it is now.